src/test/java/com/wikia/webdriver/testcases/mercurytests/SearchTests.java

```diff
@@ -35,6 +38,38 @@ public void mercury_search_navigateToPageUsingSearchSuggestions() {
     Assertion.assertTrue(driver.getCurrentUrl().equals(suggestionLink));
   }
 
+  @InBrowser(emulator = Emulator.GOOGLE_NEXUS_5)
+  @Test(groups = "mercury_search_navigateUsingSearchSuggestionsOnMobile")
+  public void mercury_search_navigateUsingSearchSuggestionsOnMobile() {
+    String suggestionLink =
+        new ArticlePage()
+            .open(MercurySubpages.MAIN_PAGE)
+            .getTopBar()
+            .openSearch()
+            .typeInSearch(this.SEARCH_PHRASE)
+            .selectSearchSuggestion(0);
+
+    Assertion.assertTrue(driver.getCurrentUrl().equals(suggestionLink));
+    Assertion.assertTrue(new SkinHelper(driver).isSkin(Skin.MERCURY));
+  }
+
+  @Execute(onWikia = MercuryWikis.MEDIAWIKI_119)
+  @Test(groups = "mercury_search_navigateUsingSearchSuggestionsOnDesktop")
+  @InBrowser(browser = Browser.FIREFOX, browserSize = "1920x1080")
+  public void mercury_search_navigateUsingSearchSuggestionsOnDesktop() {
+    Search suggestionLink =
+        new ArticlePage()
+            .open("/d/")
+            .getTopBar()
+            .openSearch()
+            .typeInSearch(this.SEARCH_PHRASE)
+            .selectSearchSuggestion(0);;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this empty statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)